sys/shell/shell.c

```diff
@@ -71,18 +71,21 @@
 #define ESCAPECHAR '\\'
 #define BLANK ' '
 
-enum PARSE_STATE {
-    PARSE_SPACE,
-    PARSE_UNQUOTED,
-    PARSE_SINGLEQUOTE,
-    PARSE_DOUBLEQUOTE,
-    PARSE_ESCAPE_MASK,
-    PARSE_UNQUOTED_ESC,
-    PARSE_SINGLEQUOTE_ESC,
-    PARSE_DOUBLEQUOTE_ESC,
+int PARSE_ESCAPE_MASK = 0b100;
```

Comment:
Why not keep this as a enum or a define? A variable does not seem needed.

Reply:
The reason I've moved this out of the enum is that the enum definition would mix real states with an escape mask for 3 of these states. I think it should be seperated from each other, so I'll rewrite it as define.